Examples/GenerativeAICLI/Sources/GenerateContent.swift

```diff
@@ -117,10 +117,8 @@ struct GenerateContent: AsyncParsableCommand {
   func modelNameOrDefault() -> String {
     if let modelName = modelName {
       return modelName
-    } else if imageURL != nil {
-      return "gemini-1.0-pro-vision-latest"
     } else {
-      return "gemini-1.0-pro"
+      return "gemini-1.5-pro-latest"
     }
```

Comment:
This cleanup is a nice example of one of the benefits of Gemini 1.5 over the older models.